core/src/epicli/data/common/ansible/playbooks/roles/helm_charts/files/system/node-exporter/Chart.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1
+appVersion: 1.0.1
+description: Prometheus exporter for hardware and OS metrics exposed by UNIX kernels, with pluggable metric collectors.
+name: node-exporter
+version: 1.1.2
+keywords:
+  - prometheus
+  - node-exporter
+  - monitoring
+home: https://github.com/bitnami/charts/tree/master/bitnami/node-exporter
+icon: https://bitnami.com/assets/stacks/node-exporter/img/node-exporter-stack-220x234.png
+sources:
+  - https://github.com/bitnami/bitnami-docker-node-exporter
+  - https://github.com/prometheus/node_exporter
+  - https://prometheus.io/
+maintainers:
+  - name: Bitnami
```

Comment:
The node-exporter Helm chart is an external project (maintained by Bitnami) and I would prefer to not include its sources.
My proposal is to just use Bitnami repo to install the chart (assuming we have Internet access in AKS/EKS case).

If we will need to provide the chart for offline mode, then we can add `charts` section to requirements.txt and download the chart in a similar way to OS packages.

Reply:
True, as We already talked - approach changed. Added bitnami repo to our local helm and all necessary changes.

Done :)